health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewHierarchy.js

```
@@ -545,9 +576,28 @@
+                                  <div style={{ display: "flex", justifyContent: "flex-end", alignItems: "center", width: "100%" }}>
+                                    <div style={{ display: "flex", justifyContent: "space-between", alignItems: "center", maxWidth: "20rem", gap:"1rem" }}>
+                                      <div style={{ marginRight: "10px" }}>
+                                        {fileName}
+                                      </div>
+                                      <div className="dustbin-icon" onClick={() => removeFile()} style={{ cursor: "pointer", marginTop:"1.15rem" }}>
+                                        <DustbinIcon />
+                                      </div>
+                                    </div>
+                                  </div>
+                                }
                                 <div style={{height:"2rem"}}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)